ansible/roles/daliuge_runtime/tasks/runtime.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Install developments tools envrironment group
+  yum:
+    name: "@Development Tools"
+    state: installed
```

Comment:
Small nit on consistency - you use 'present' below, I believe they have the same effect.